### PR TITLE
Cyclotron/Hat Lights

### PR DESCRIPTION
**Pull request by DustinGrau**
Adding content and photos regarding the installation of the hat lights on the wand, and cyclotron lid lights on the pack. I've also switched the HOSE.md from being an optional doc to now being the description of the kit wand loom replacements (plus the original DIY solution).

Also added new STL specific to the Dayton Audio DMA80 speaker: This is an alternate supporting mount for that speaker (when mounted to the top of the pack), while the original is named with a "_Generic" suffix for the 3" speakers already proposed for use.

### CODE REVIEW

File: HOSE.md [hunk 1]
```diff
@@ -1,6 +1,47 @@
...
-## Hose Example #1
+In order to deliver both power and data signals to the wand, a new set of cabling is required. As wiring is typically desired to remain internal to the hose the following approaches have been used to replace the stock 2-wire (power-only) hose with a new 4-wire solution. The existing wire loom (black hose) can be reused if desired, or replaced with a more flexible brand.
+
+## PCB Kit Hose
+
+For those who have purchased the PCB kits with the hose replacement, the following parts are utilized:
+
+* 1 - [CNLINKO 4-Pin Connector](https://a.co/d/8ALa9Gq) - Connector for the pack which uses a square connector roughly the size of the stock connection
+* 1 - [Neutrik NL4FX 4-Pole Locking SpeakON Cable Connector](https://a.co/d/9c7djjo) - This is the connector at the wand end of the hose
+* 1 - [Neutrik NL4MP 4-Pole SpeakON Panel Mount](https://a.co/d/atMuNfz) - This is the socket for the wand itself
```
Comment (gpstar81) [inline HOSE.md:11]: The NL4MP-ST is the one we are going to be providing. It has screw terminals and requires no soldering.

https://www.neutrik.com/en/product/nl4mp-st

**Review by gpstar81 — COMMENTED** [review]
I should mention, the hose kits will be coming pre-assembled. The cnlinkos will be pre-soldered already with all wires attached. Also we are providing already modded Neutrik NL4MP-ST (screw terminal version) for the wand handle and the NL4FC for the hose side. They both have screw terminals so nobody has to solder anything.

I can add some photos of the wire kits and update some of the text to reflect it if you want me to? Or keep your existing changes for another section of self building a hose kit?

**Review by gpstar81 — APPROVED** [review]

**Comment by DustinGrau** [comment]
> I can add some photos of the wire kits and update some of the text to reflect it if you want me to? Or keep your existing changes for another section of self building a hose kit?

I was going to email you before I set up the request for a review, but you beat me to it. This was mostly me getting a brain-dump out there for the DIY approach as I expected there would be some changes for the kit versions. But to answer your question, yes if you want to edit/add the content specific to the kits that might be better as I'm not entirely up to date on your plan for those. You could just make mine a secondary DIY approach then if I don't have the right parts--I honestly thought I had the right ones listed, but went with what I had found on Amazon.

**Comment by gpstar81** [comment]
> > I can add some photos of the wire kits and update some of the text to reflect it if you want me to? Or keep your existing changes for another section of self building a hose kit?
> 
> 
> 
> I was going to email you before I set up the request for a review, but you beat me to it. This was mostly me getting a brain-dump out there for the DIY approach as I expected there would be some changes for the kit versions. But to answer your question, yes if you want to edit/add the content specific to the kits that might be better as I'm not entirely up to date on your plan for those. You could just make mine a secondary DIY approach then if I don't have the right parts--I honestly thought I had the right ones listed, but went with what I had found on Amazon.

No worries. I made some minor changes to several other documentation pages and added your excellent videos. I will do up a wire kit instruction page over the coming days.

**Pull request merged**